src/transformers/models/videomae/feature_extraction_videomae.py

```diff
@@ -97,6 +97,9 @@ def normalize_video(self, video, mean, std):
 
         return (video - mean[None, :, None, None]) / std[None, :, None, None]
 
+    def to_numpy_array_video(self, video, rescale=None, channel_first=True):
+        return [self.to_numpy_array(frame, rescale, channel_first) for frame in video]
```

Comment:
Also a bit unrelated: slightly confusing to me that `to_numpy_array` does things like rescaling and transposing, e.g. I wouldn't have expected:
```
image = image.transpose(2, 0, 1)   
```
or 
```
image = image.astype(np.float32) / 255.0
```
to be in a function that is called `to_numpy_array`

Reply:
Similarly isn't `make_channel_first` always `False` given that it has already been done before?

Reply:
> Also a bit unrelated: slightly confusing to me that to_numpy_array does things like rescaling and transposing e.g. I wouldn't have expected:

I completely agree! As part of the work replacing feature extractors with image processors methods like `to_numpy_array` and `normalize` are being reworked so logic like rescaling is taken out.